src/components/editor/EditAuthorsForm.jsx

```diff
@@ -1,80 +1,37 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 import BaseComponent from 'lib/BaseComponent';
 import _ from 'lodash';
 
 export default class EditAuthorsForm extends BaseComponent {
   constructor(props) {
     super(props);
     this.safeSetState({
-      // Key is slug, value is name
-      authorsAdded: {},
-      // An object where key is slug and value is whether it's deleted or not
-      authorsDeleted: _.reduce(this.props.data, (cur, next) => {
-        cur[next.slug] = false;
-        return cur;
-      }, {})
+      addAuthorValue: ""
     });
     this.handleAddAuthor = this.handleAddAuthor.bind(this);
   }
 
   componentDidUpdate() {
-    this.props.onChange(this.state.authorsAdded, this.state.authorsDeleted);
+    this.props.onChange(ReactDOM.findDOMNode(this).parentNode);
```

Comment:
So: The way I structured it right now I get the data from the form by having the DOM formNode which is usually passed by the parent of the event target, and then from here you can do formNode[fieldName].value to get the values. So when I can't trigger the event, I wanted to find the formNode, and this was then how I found the formNode. 

Can you see a better way? I wouldn't be surprised if there was one, but this is what I found that I thought worked okay.